packages/performant-ui/src/components/PageStats.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useMemo } from 'react'
+
+interface Props {
+  perPage: number,
+  page: number,
+  itemCount: number
+}
+
+const PageStats: React.FC<Props> = (props) => {
+  const start = (props.perPage * (props.page - 1)) + 1
+  const end = Math.min(start + props.perPage - 1, props.itemCount)
+
+  return (
+    <p className='text-sm font-sans'>
+      Showing&nbsp;
```

Comment:
How are we handling i18n for this package?

Reply:
Agree that we should make strings internationalizable as we go, if possible. If not, we should open an issue now to track strings to be factored out and translated.

Reply:
I've filed an issue for i18n: https://github.com/performant-software/react-components/issues/547

I don't think this should be a blocker - I just met with Nick and Lorin and they were pretty clear that we need to get this package out ASAP.